src/arithmetic/algebraic_expression.c

```diff
@@ -123,6 +104,41 @@ AlgebraicExpression **_AlgebraicExpression_Intermidate_Expressions(AlgebraicExpr
     return expressions;
 }
 
+/* Return index of matrix operand within algebraic expression
+ * which has the least number of none zero values. */
+int _AlgebraicExpression_MinMatPos(const AlgebraicExpression *ae) {
+    int minMatPos = 0;  // Position of matrix within ae operands.
+    GrB_Index nvals;    // number of NNZ.
+    GrB_Index minNvals; // minimum number of NNZ.
+    
+    GrB_Matrix M = ae->operands[0].operand;
+    GrB_Matrix_nvals(&minNvals, M);
```

Comment:
Initializing `minNvals` to `GrB_Index` max would let us delete these 2 lines and start the following loop at index 0.